test/gtest/lstm_extra.cpp

```diff
@@ -109,48 +107,62 @@ TEST_P(ConfigWithFloat, FloatTest)
-    {flags + commonFlags + " -dir-mode 0 --no-cy --no-dcx"},
-    {flags + commonFlags + " -dir-mode 1 --no-hy"},
-    {flags + commonFlags + " -dir-mode 1 --no-dhx"},
-    {flags + commonFlags + " -dir-mode 1 --no-hy --no-dhx"},
-    {flags + commonFlags + " -dir-mode 1 --no-cy"},
-    {flags + commonFlags + " -dir-mode 1 --no-hy --no-cy"},
-    {flags + commonFlags + " -dir-mode 1 --no-dcx"},
-    {flags + commonFlags + " -dir-mode 1 --no-cy --no-dcx"},
-    {flags + commonFlags + " -dir-mode 0 --no-hx --no-dhy --no-cx --no-dcy --no-hy --no-dhx --no-cy --no-dcx"},
-    {flags + commonFlags + " -dir-mode 1 --no-hx --no-dhy --no-cx --no-dcy --no-hy --no-dhx --no-cy --no-dcx"}
-        // clang-format on
-    };
+        {flags + commonFlags + " -dir-mode 0 --no-hx"},
+        {flags + commonFlags + " -dir-mode 0 --no-dhy"},
+        {flags + commonFlags + " -dir-mode 0 --no-hx --no-dhy"},
+        {flags + commonFlags + " -dir-mode 0 --no-cx"},
+        {flags + commonFlags + " -dir-mode 0 --no-hx --no-cx"},
+        {flags + commonFlags + " -dir-mode 0 --no-dcy"},
+        {flags + commonFlags + " -dir-mode 0 --no-cx --no-dcy"},
+        {flags + commonFlags + " -dir-mode 1 --no-hx"},
+        {flags + commonFlags + " -dir-mode 1 --no-dhy"},
+        {flags + commonFlags + " -dir-mode 1 --no-hx --no-dhy"},
+        {flags + commonFlags + " -dir-mode 1 --no-cx"},
+        {flags + commonFlags + " -dir-mode 1 --no-hx --no-cx"},
+        {flags + commonFlags + " -dir-mode 1 --no-dcy"},
+        {flags + commonFlags + " -dir-mode 1 --no-cx --no-dcy"},
+        {flags + commonFlags + " -dir-mode 0 --no-hy"},
+        {flags + commonFlags + " -dir-mode 0 --no-dhx"},
+        {flags + commonFlags + " -dir-mode 0 --no-hy --no-dhx"},
+        {flags + commonFlags + " -dir-mode 0 --no-cy"},
+        {flags + commonFlags + " -dir-mode 0 --no-hy --no-cy"},
+        {flags + commonFlags + " -dir-mode 0 --no-dcx"},
+        {flags + commonFlags + " -dir-mode 0 --no-cy --no-dcx"},
+        {flags + commonFlags + " -dir-mode 1 --no-hy"},
+        {flags + commonFlags + " -dir-mode 1 --no-dhx"},
+        {flags + commonFlags + " -dir-mode 1 --no-hy --no-dhx"},
+        {flags + commonFlags + " -dir-mode 1 --no-cy"},
+        {flags + commonFlags + " -dir-mode 1 --no-hy --no-cy"},
+        {flags + commonFlags + " -dir-mode 1 --no-dcx"},
+        {flags + commonFlags + " -dir-mode 1 --no-cy --no-dcx"},
+        {flags + commonFlags +
+         " -dir-mode 0 --no-hx --no-dhy --no-cx --no-dcy --no-hy --no-dhx --no-cy --no-dcx"},
+        {flags + commonFlags +
+         " -dir-mode 1 --no-hx --no-dhy --no-cx --no-dcy --no-hy --no-dhx --no-cy --no-dcx"}};
 
     return test_cases;
 }
 
-INSTANTIATE_TEST_SUITE_P(ConvTrans, ConfigWithFloat, testing::Values(GetTestCases()));
+INSTANTIATE_TEST_SUITE_P(LstmExtra, ConfigWithFloat, testing::Values(GetTestCases("--float")));
+
+INSTANTIATE_TEST_SUITE_P(LstmExtra, ConfigWithHalf, testing::Values(GetTestCases("--half")));
```

Comment:
[Notice] HALF is not necessary here, but again, I am not against this. Sorry for confusion

/cc @shurale-nkn

Reply:
[resolved]